app/assets/stylesheets/notFound.scss

```diff
@@ -0,0 +1,37 @@
+@import "variables";
+
+.not-found-main{
+
+    h1{
+        margin-top: 2rem;
+        color: $blue-purple;
+        text-align: center;
+        font-size: 7rem;
+        font-weight: bolder;
+    }
+
+    h5{
+        text-align: center;
+        color: $blue-purple;
+        margin-bottom: 5rem;
+    }
+
+    p{
+        text-align: center;
+        color: $blue-purple;
+        font-size: 2rem;
+    }
```

Comment:
Not sure if you want these to apply across all h1, h5, and p tags, but one thing y'all could do is add class names for all your title or subtitles or body text in your application.scss, and then just apply the class name to elements on each page rather than redefining similar style rules across componenets.